src/components/Node/Node.js

```diff
@@ -414,6 +415,7 @@ const ThemedContent = styled.div`
   ${({ theme }) => theme.nodeFlexDirection === "row" && `padding-left: 4px`};
   ${({ theme }) => theme.nodeFlexDirection === "column" && `padding-top: 4px`};
   .label {
+    // word-break: break-all;
```

Comment:
what about long names?

Reply:
Yes, don't know, I don't like word-break as it will break randomly

Reply:
would you rather have a name that gets out of the node bounds?

Reply:
Yes